core/src/main/java/org/apache/calcite/util/BuiltInMethod.java

```diff
@@ -315,6 +316,8 @@ public enum BuiltInMethod {
   FP_DECIMAL_CAST(Primitive.class, "fpDecimalCast",
       Number.class, int.class, int.class),
   INTEGER_CAST(Primitive.class, "integerCast", Primitive.class, Object.class),
+  INTEGER_CAST_ROUNDING_MODE(Primitive.class, "integerCast",
```

Comment:
The other cast method should probably be removed.
but many other cast methods have to be considered, including DECIMAL to DECIMAL, FP to integer, FP to decimal, DECIMAL to FP and perhaps even string to decimal

Reply:
Yes. According to your advice, I found some places that use the default `RoudingMode.DOWN`. I'll see what I can do.